### ISSUE TITLE
BDSMLR error invalid !

### ISSUE THREAD

**Issue by berat1397** [KurtBestor/Hitomi-Downloader#2676]
I'm gettin many process this error.

[alrightyougotmensfw.bdsmlr.txt](https://github.com/KurtBestor/Hitomi-Downloader/files/5469947/alrightyougotmensfw.bdsmlr.txt)
[sweetcages.bdsmlr.txt](https://github.com/KurtBestor/Hitomi-Downloader/files/5469948/sweetcages.bdsmlr.txt)

**Comment by KurtBestor** [comment]
Try this: https://github.com/KurtBestor/Hitomi-Downloader/releases/tag/Technical-Preview

**Comment by berat1397** [comment]
Yes. İt is working. Thank you for your help. Why isn't working other release ?

**Comment by KurtBestor** [comment]
I've fixed this now on the beta version.

It will be applied in the next release.

**Comment by berat1397** [comment]
Thank you.
